src/Symfony/Component/Serializer/Mapping/Loader/AnnotationLoader.php

```diff
@@ -65,6 +66,8 @@ public function loadClassMetadata(ClassMetadataInterface $classMetadata): bool
                     $annotation->getTypeProperty(),
                     $annotation->getMapping()
                 ));
+            } elseif ($annotation instanceof Groups) {
```

Comment:
Nitpicking: use `match` instead? Or at list use `continue` at the end of the first `if` to remove the else and reduce a bit the complexity?

Reply:
@dunglas done, replaced `if` construct  with `match`
https://github.com/symfony/symfony/pull/49594/commits/5fafefb42044dca23909ac4251bae40f6597820a